src/PrestaShopBundle/Controller/Admin/Improve/Design/MailThemeController.php

```diff
@@ -117,7 +118,7 @@ public function generateMailsAction(Request $request)
                 $modulesMailFolder = '';
                 //Overwrite theme folder if selected
                 if (!empty($data['theme'])) {
-                    $themeFolder = $this->getParameter('themes_dir') . '/' . $data['theme'];
+                    $themeFolder = $themes_dir . '/' . $data['theme'];
```

Comment:
I might be wrong but it sounds like the original intent was
```suggestion
                    $themeFolder = $this->container->getParameter('themes_dir') . '/' . $data['theme'];
```

Reply:
It was but it seems it doesn't work anymore, because the `parameter_bag` is not accessible in the container anymore. See the error thrown here: https://github.com/symfony/symfony/blob/2ea14ffc32bc46319bff29ab6ccaeea5ba5aa9f0/src/Symfony/Bundle/FrameworkBundle/Controller/AbstractController.php#L68

So maybe there is more simple, the controller would need to be registered as a service or maybe we need to configure the `parameter_bag` in our container, either way, this might be doable via configuration in the framework instead of modifying the code.

Reply:
Is it really needed to register a parameterBag for the container for this method which is only requires something already defined in service? It will use memory for nothing, seems a little bit overkill imho